### PR TITLE
Fix: Removed Hardcoded text from XML Files

### LINKED ISSUE #624: Remove Hardcoded Text
**Summary:**

There are some text that are hard-coded in the XML files of the project, remove them by extracting them as string resources.

**Expected behavior:**

There should be no hard-coded text.

### PR DESCRIPTION
**Pull request by rachittt**
Fixes #624 

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/res/layout/activity_merchant_transaction.xml [hunk 1]
```diff
@@ -57,53 +57,54 @@
...
+                    android:src="@drawable/logo_pnb" />
 
                 <TextView
                     android:id="@+id/tv_pay_to_name"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:layout_gravity="center"
                     android:layout_marginTop="@dimen/marginItemsInSectionSmall"
-                    android:text="Naman Dwivedi 2"
+                    android:text="@string/naman_dwivedi_2"
```
Comment (luckyman20) [inline mifospay/src/main/res/layout/activity_merchant_transaction.xml]: Change it to something like `test_user` and then enter some generic name.

File: mifospay/src/main/res/layout/activity_merchant_transaction.xml [hunk 2]
```diff
@@ -57,53 +57,54 @@
...
+                    android:textSize="@dimen/text_large" />
 
                 <TextView
                     android:id="@+id/tv_pay_to_vpa"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:layout_gravity="center"
                     android:layout_marginTop="@dimen/marginItemsInSectionSmall"
-                    android:text="naman.dwivedi2@mifos"
+                    android:text="@string/naman_dwivedi_email"
```
Comment (luckyman20) [inline mifospay/src/main/res/layout/activity_merchant_transaction.xml]: Same as above. Also, this is not email ID, it is UPI ID also known as Virtual Private Address.

File: mifospay/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -235,4 +238,10 @@
     <string name="amount">Amount</string>
     <string name="merchant_history_message">The transaction history with this merchant will be shown here</string>
     <string name="log_out_title">Do you want to logout?</string>
+    <string name="transfer_money_to_this_merchant">Do you want to logout?</string>
```
Comment (luckyman20) [inline mifospay/src/main/res/values/strings.xml]: `name` here seems irrelevant.

File: mifospay/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -235,4 +238,9 @@
     <string name="amount">Amount</string>
     <string name="merchant_history_message">The transaction history with this merchant will be shown here</string>
     <string name="log_out_title">Do you want to logout?</string>
+    <string name="test_user">Naman Dwivedi 2</string>
```
Comment (luckyman20) [inline mifospay/src/main/res/values/strings.xml]: Add some generic name here instead of `Naman Dwivedi 2`.

Comment (rachittt) [reply mifospay/src/main/res/values/strings.xml]: okay

File: mifospay/src/main/res/layout/activity_merchant_transaction.xml [hunk 5]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (luckyman20) [inline mifospay/src/main/res/layout/activity_merchant_transaction.xml]: Why did you add this?

File: mifospay/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -235,4 +238,10 @@
     <string name="amount">Amount</string>
     <string name="merchant_history_message">The transaction history with this merchant will be shown here</string>
     <string name="log_out_title">Do you want to logout?</string>
+    <string name="transfer_money_to_this_merchant">Transfer money to this merchant</string>
+    <string name="test_user">Dwight Schrute</string>
+    <string name="test_vpa">naman.dwivedi2@mifos</string>
```
Comment (luckyman20) [inline mifospay/src/main/res/values/strings.xml]: Change this too.

**Comment by rachittt** [comment]
> Have you checked java files too? Are you sure they don't have any hardcoded string?

Yes I did. There were no hardcoded strings in java files :)

**Comment by luckyman20** [comment]
@rachittt Thank you for your contributions. Excellent work.
